pkg/kritis/metadata/containeranalysis/containeranalysis.go

```diff
@@ -128,29 +129,53 @@ func (c Client) fetchAttestationOccurrence(containerImage string, kind string, a
 		return nil, fmt.Errorf("%s is not a valid image hosted in GCR", containerImage)
 	}
 
+	noteName := fmt.Sprintf("%s/notes/%s", auth.Spec.NoteReference, auth.Name)
 	req := &grafeas.ListNoteOccurrencesRequest{
-		Name: auth.Spec.NoteReference,
-		// TODO: re-add `kind` clause in filter
-		//       (currently not supported as per https://github.com/grafeas/kritis/issues/401#issuecomment-538947608)
+		Name: noteName,
```

Comment:
As per comment above, I suggest reverting the change to this line.

Reply:
Let's do this as part of the larger change to have consistency with BinAuthz on how attestors represent the note.